### PR TITLE
Refactored load to xse taking into consideration more data cases

### PR DESCRIPTION
**Pull request by KarenAJ**

### CODE REVIEW

File: app/core/management/commands/load_metadata_into_xse.py [hunk 1]
```diff
@@ -92,21 +92,22 @@ def post_data_to_xse(data):
...
+        supplemental_data = {k: v for k, v in row['metadata'][
+            'Supplemental_Ledger'].items() if v != "NaT" and v and v != "null"}
+        composite_ledger_dict = {"Supplemental_Ledger": supplemental_data}
     for item in row['metadata']['Metadata_Ledger']:
         # Removing empty/Null data fields in metadata to be sent to XSE
         for item_nested in row['metadata']['Metadata_Ledger'][item]:
-            if not row['metadata']['Metadata_Ledger'][item][item_nested]:
+            if not row['metadata']['Metadata_Ledger'][item][item_nested] or \
+                    row['metadata']['Metadata_Ledger'][item][item_nested] == \
+                    "NaT" or \
```
Comment (mattdrayer) [inline app/core/management/commands/load_metadata_into_xse.py:106]: "NaT"?

Comment (KarenAJ) [reply app/core/management/commands/load_metadata_into_xse.py:106]: Yup, this is a unique problem from AETC data where missing date values are assigned "NaT". This is causing the datatype match problem for the elastic search load process. @mattdrayer

**Review by SHGODE — APPROVED** [review]
Approved

**Review by mattdrayer — APPROVED** [review]
Looks good!

**Pull request merged**